src/main/java/com/github/splendor_mobile_game/game/model/Game.java

```diff
@@ -78,17 +85,50 @@ public boolean startGame(Room room) {
         this.onyxTokens     = createTokenList(TokenType.ONYX);
         this.goldTokens     = createTokenList(TokenType.GOLD_JOKER);
 
+        //If startGame will be called second time, we need to clear decks
+        //Maybe will be good idea to make startGame private and call it in constructior?
+        //because why we would call startGame more than once?
+        decks.get(CardTier.LEVEL_1).clear();
+        decks.get(CardTier.LEVEL_2).clear();
+        decks.get(CardTier.LEVEL_3).clear();
+        revealedDecks.get(CardTier.LEVEL_1).clear();
+        revealedDecks.get(CardTier.LEVEL_2).clear();
+        revealedDecks.get(CardTier.LEVEL_3).clear();
```

Comment:
We won't call it ever again on the same object. We don't need to clear `decks` and `revealedDecks`